### DISCUSSION TITLE
Improving LLM Output Quality With Masked Output: Seeking Collaborators!

### DISCUSSION THREAD

**Discussion by Mrw33554432** [FasterDecoding/Medusa#58] [General]
Hello everyone!

I've been brainstorming some concepts on enhancing the output quality of LLMs. The core of the idea revolves around masked output, and I've already laid the groundwork for dataset generation and the basic training process.

However, due to my current work commitments, I'm unable to bring this idea to fruition by myself. I believe that with collective effort, we can achieve significant progress on this.

Is there a real-time chat platform, similar to Discord, where we can discuss this further? I'm eager to collaborate and share more details with interested individuals.

Looking forward to your responses and potential collaborations!

**Comment by leeyeehoo** [comment]
Sorry just saw this! You can drop me an email if you like :)
